site/_config.yml

```diff
@@ -34,6 +34,7 @@ hello:
   nl: Hallo!
   he: "שלום!"
   ko: 안녕하세요!
+  zh-CN: 你好！
```

Comment:
> Note: I use `zh-CN` as the corresponding i18n code but not `zh`

That's totally fine and correct here. Many other countries will use a country-specific designation as part of their browser configuration (for example Portuguese in portugal and brazil are considered different as well and those countries will favor `pt-PT` or `pt-BR`).

worth calling out the in readme for users of these language codes.